packages/webex/package.json

```diff
@@ -46,7 +44,7 @@
     "test:integration": "webex-legacy-tools test --integration --runner mocha",
     "test:style:todo": "eslint ./src/**/*.*",
     "test:unit:todo": "webex-legacy-tools test --unit --runner jest",
-    "deploy:npm": "echo yarn npm publish"
+    "deploy:npm": "yarn npm publish"
```

Comment:
@sreenara - Has Tim checked out everything and cleared path for this?

Reply:
@mkesavan13 Tim had asked us to test the pipeline by doing some commits, and checking if commits that have dependent packages cause :
* Tests to run on dependent packages as well as changed package
* Sync and increment versions on dependent package as well as changed package
* Publish dependent as well as changed package.

The commits we have been doing for calling package over the last couple of weeks have helped us test and ensure the tooling is running correctly. This is why we've removed the echo. In any case, the beta period for calling and meeting will provide the soak time for testing the publish as well to the next branch. So we should be good.

Reply:
Got it. Thanks for the response @sreenara